app/code/Magento/Backend/Ui/Component/Control/SaveSplitButton.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Backend\Ui\Component\Control;
+
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+use Magento\Ui\Component\Control\Container;
+
+/**
+ * @api
```

Comment:
`@api` classes should be descriptive, isn't it? Something like "Represents split-button with bla-bla-bla options".

Reply:
Fixed